README.md

```diff
@@ -20,6 +20,7 @@ npm install react-native-unimodules
 
 - Go back to the `ios` directory and open your Podfile, make your Podfile look [like this one](https://gist.github.com/sjchmiela/6c079f2173938a9a61a7c6f053c45000).
   - If you need to customize the path to node_modules, for example because you are using yarn workspaces, then you can pass in a param for this: `use_unimodules!(modules_paths: ['./path/to/node_modules'])`
+  - If you need to exclude some of the default modules that you are not using, then you can pass in a param for this: `use_unimodules!(exclude: ['unimodules-face-detector-interface'])`
```

Comment:
```suggestion
  - If you need to exclude some of the unimodules that you are not using but they got installed by your other dependencies (like `expo`), then you can pass in `exclude` param for this. For example, if you want to exclude `expo-face-detector`, you may want to use this: `use_unimodules!(exclude: ['expo-face-detector'])`
```

I wouldn't call them `default modules` - they are default in Expo context, but not in react-native-unimodules context. Also, excluding interfaces is generally bad idea because interfaces are a dependencies of other unimodules. Quick example:
`expo-camera` depends on `unimodules-face-detector-interface` but not on `expo-face-detector` that implements this interface. If you installed any unimodule that implements `unimodules-face-detector-interface` (it doesn't have to be `expo-face-detector`), then it will be returned for use by `expo-camera`.
If there is no such module implementing that interface, then Camera can throw an error in runtime, but only when you use a prop or a method that refers to face detector.

So, excluding an interface here might cause build errors because other unimodules may depend on them. However, it should be safe to exclude non-interfaces unimodules (other than `@unimodules/core` and adapters) as nothing else depend on them directly.